README.md

```diff
@@ -27,6 +27,9 @@ While the following function signatures are probably self-explanatory, you can q
 *   `δ(j₁, j₂, j₃) -> ::Bool`
 *   `Δ(T::Type{<:AbstractFloat} = Float64, j₁, j₂, j₃) -> ::T`
 
+The package also defines the `HalfInteger` type that can be used to represent half-integer values.
```

Comment:
I wasn't sure if it should also be exported or not. I am leaning towards not, since it's not really core WignerSymbols functionality necessarily.